editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -831,18 +831,29 @@ void Node3DEditorViewport::_find_items_at_pos(const Point2 &p_pos, Vector<_RayRe
 	Vector<ObjectID> instances = RenderingServer::get_singleton()->instances_cull_ray(pos, pos + ray * camera->get_far(), get_tree()->get_root()->get_world_3d()->get_scenario());
 	HashSet<Node3D *> found_nodes;
 
+	Node *edited_scene = get_tree()->get_edited_scene_root();
+
 	for (int i = 0; i < instances.size(); i++) {
 		Node3D *spat = Object::cast_to<Node3D>(ObjectDB::get_instance(instances[i]));
 
 		if (!spat) {
 			continue;
 		}
 
-		if (found_nodes.has(spat)) {
+		Node3D *item = spat;
+		if (item != edited_scene) {
+			item = Object::cast_to<Node3D>(edited_scene->get_deepest_editable_node(item));
+		}
```

Comment:
Some of the other methods handle group stuff here, but `_list_select` has its own group processing code so not touching it here, unsure what's best here, leaving as is for now